src/pages/data-reporting/reports-api/v2.0/index.mdx

```diff
@@ -7,6 +7,7 @@ tagline: "Access and extract data from Reports."
 desc: "See how exactly you can access and extract data from Reports with the use of Reports API."
 versionGroup: "data-reporting"
 apiVersion: "2.0"
+robots: "noindex"
 ---
 
 # Introduction
```

Comment:
This document can be indexed because it ha significantly different content than other versions